Gemfile.lock

```diff
@@ -261,14 +261,14 @@ GEM
       rspec-mocks (~> 3.4.0)
       rspec-support (~> 3.4.0)
     rspec-support (3.4.1)
-    rubocop (0.37.2)
-      parser (>= 2.3.0.4, < 3.0)
+    rubocop (0.40.0)
```

Comment:
@bdunne (no problem, happy to help, I wanted to get a few features in)  I guess the question that I'm trying to see (so I can understand the Parser errors) is how can I generate them locally (I haven't see any errors, which is why I ask).  Again, happy to go down to 2.2.4, I'm just trying to understand what the problem is on my end.  Thanks for your helping me understand this.